guides/source/action_view_helpers.md

```diff
@@ -5,25 +5,33 @@ Action View Helpers
 
 After reading this guide, you will know:
 
-* How to format dates, strings, and numbers
-* How to link to images, videos, stylesheets, etc...
-* How to sanitize content
-* How to localize content
+* How to format dates, strings, and numbers.
+* How to link to images, videos, stylesheets, etc.
+* How to work with Atom feeds and JavaScript in the views.
+* How to cache, capture, debug and sanitize content.
+* How to work with text and tags.
 
 --------------------------------------------------------------------------------
 
 Overview of Helpers Provided by Action View
 -------------------------------------------
 
-WIP: Not all the helpers are listed here. For a full list see the [API documentation](https://api.rubyonrails.org/classes/ActionView/Helpers.html)
-
-The following is only a brief overview summary of the helpers available in Action View. It's recommended that you review the [API Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers.html), which covers all of the helpers in more detail, but this should serve as a good starting point.
+The following outlines **some of the most commonly used helpers** available in
+Action View. It's recommended that you review the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers.html),
+which covers all of the helpers in more detail, but this should serve as a good
+starting point.
 
 ### AssetTagHelper
 
-This module provides methods for generating HTML that links views to assets such as images, JavaScript files, stylesheets, and feeds.
+This module provides methods for generating HTML that links views to assets such
+as images, JavaScript files, stylesheets, and feeds.
 
-By default, Rails links to these assets on the current host in the public folder, but you can direct Rails to link to assets from a dedicated assets server by setting [`config.asset_host`][] in the application configuration, typically in `config/environments/production.rb`. For example, let's say your asset host is `assets.example.com`:
+By default, Rails links to these assets on the current host in the public
+folder, but you can direct Rails to link to assets from a dedicated assets
+server by setting [`config.asset_host`][] in the application configuration,
```

Comment:
Maybe something for the Ruby on Rails Guides Guidelines:

I noticed that sometimes we add the link directly inline, for example, this could have been [`config.asset_host`](configuring.html#config-asset-host).

but at other times we reference the name and bunch the links at the bottom of the section, like in this case:

```
[`config.asset_host`]: configuring.html#config-asset-host
```

Maybe we could chat about which is preferable, or if there's certain circumstances where you'd use one over the other and try to keep it consistent.

Reply:
I prefer it inline, I feel like there's less overhead, it's easier to find and change, and potentially has less room for error. However, I too don't feel too strongly about it and would be okay with either way as long as it's consistent.

Reply:
Right, that's a good point... it's easier to maintain / less error prone if they're inline, even if that makes it very long sometimes.

@bhumi1102 any thoughts? (cc @p8)